pkg/ipv6util/ipv6util.go

```diff
@@ -157,3 +181,45 @@ func AreAddressesIPv6(addrs []string) bool {
 	}
 	return isIpv6
 }
+
+func parseIPAddressInPxctlServiceKvdbEndpoints(kvdbEndpointsOutput string) ([]string, error) {
+	kvdbEndPts := kvdbEndPtsRgx.FindAllSubmatch([]byte(kvdbEndpointsOutput), -1)
+
+	kvdbEndPtsIPs := []string{}
+	for _, endPt := range kvdbEndPts {
+		ip, _, err := net.SplitHostPort(string(bytes.TrimSpace(endPt[1])))
+		if err != nil {
+			return kvdbEndPtsIPs, fmt.Errorf("Endpoint parse error %v", err)
+		}
+		kvdbEndPtsIPs = append(kvdbEndPtsIPs, ip)
+	}
+	return kvdbEndPtsIPs, nil
+}
+
+func parseIPAddressInPxctlServiceKvdbMembers(kvdbMembersOutput string) ([]string, error) {
+	kvdbMemberIPs := []string{}
+	for _, line := range strings.Split(kvdbMembersOutput, "\n") {
+		if strings.Contains(line, "http") {
+			cols := strings.Fields(strings.TrimSpace(line))
+			if len(cols) >= 2 {
+				endPt := kvdbEndPtsRgx.FindSubmatch([]byte(strings.Trim(cols[2], "[]")))
+				ip, _, err := net.SplitHostPort(string(bytes.TrimSpace(endPt[1])))
+				if err != nil {
+					return kvdbMemberIPs, fmt.Errorf("Member parse error %v", err)
+				}
+				kvdbMemberIPs = append(kvdbMemberIPs, ip)
+			}
+		}
+	}
+	return kvdbMemberIPs, nil
+}
+
+// ParseIPAddressInPxctlResourceDownAlert extract IP address from specific resource down alert description."
```

Comment:
similar comment:
add example on what's being parse.
nit: remove the dot and quote at the end